src/test/java/seedu/address/logic/parser/SharkieParserTest.java

```diff
@@ -96,6 +98,15 @@ public void parseCommand_list() throws Exception {
                 instanceof PeopleListCommand);
     }
 
+    @Test
+    public void parseCommand_owe() throws Exception {
+        String debt = "$4.00,2020-02-20";
+        PeopleOweCommand command = (PeopleOweCommand) parser.parseCommand(PEOPLE_COMMAND_TYPE + " "
+                + PeopleOweCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased() + " "
+                + PersonUtil.getDebtDescription(new Debt(debt)));
```

Comment:
Can consider creating a TypicalDebts in the testutil package to avoid creating new debts for all tests involving debts